### ISSUE TITLE
Folder... Loading

### ISSUE THREAD

**Issue by IMNotMax** [SENTINELITE/StreamDeck-Shortcuts#16]
First, nice work ! Really !!!
I have an issue since the last update

looks like the issue #6 but no restriction on my network (issue #7)

Folder and shortcuts only show Loading (on old and new buttons)
<img width="544" alt="image" src="https://user-images.githubusercontent.com/16235859/193052955-86bdb0e7-9520-4a62-83a0-61790e25b6c0.png">

The only action with the update is I have deleted in system preferences -> security and privacy -> privacy some strange things (not applications) with a yellow triangle showing an error. maybe one of them was the plugin ?

**Comment by SENTINELITE** [comment]
Hey, there, @IMNotMax! 

This issue appears to be related to #15. You can confirm this, if you follow the instructions [here](https://www.craft.do/s/SNZGwoLUs6uumS), & see if you get the following error, when going to customize a key, via the user interface.

```
Uncaught ReferenceError: JSON5 is not defined
    at connectElgatoStreamDeckSocket (main_pi.js:42)
    at <anonymous>:1:1
```

**Comment by IMNotMax** [comment]
Same kind of issue:


```
main_pi.js:42 Uncaught ReferenceError: JSON5 is not defined
    at connectElgatoStreamDeckSocket (main_pi.js:42)
    at <anonymous>:1:1
DevTools failed to load SourceMap: Could not load content for https://browser.sentry-cdn.com/7.13.0/bundle.min.js.map: 
```

EDIT: I tried to delete some (old and unused) shortcuts. I only have 10 active shortcuts now but the issue remains.

**Comment by SENTINELITE** [comment]
Hey, @IMNotMax!

I've submitted a patch already. Just waiting for Elgato to give the okay! Should be out tomorrow/next week, depending on their schedule. 😄

**Comment by IMNotMax** [comment]
👍

**Comment by SENTINELITE** [comment]
@IMNotMax, the update is out now. Please restart the SD Software & let me know how it goes for you! Sorry for the trouble, again! 😅

**Comment by IMNotMax** [comment]
Thanx and don’t be sorry. You’ve created a great piece of software which helps a lot a people !
I’ll restart tomorrow first thing !

**Comment by punxsutawney** [comment]
> @IMNotMax, the update is out now. Please restart the SD Software & let me know how it goes for you! Sorry for the trouble, again! 😅

@SENTINELITE I had the same problem and can confirm it works again. Thank you very much! your software is really helpful, great job.

**Comment by IMNotMax** [comment]
Confirmed ! It works (and I love the "Interface bug" label ;)

**Comment by themcclure** [comment]
Phew. I came here with the same issue. I had restarted SD a few times - but not since this morning.
I did, and now it's all working again.

Thanks!

**Comment by Coder84619** [comment]
Hmm still not working for me. I uninstall/reinstalled the Shortcuts plugin, and I rebooted my Mac (M1 Pro Monterey). Still stuck with "Loading...".

**Comment by IMNotMax** [comment]
Hi @Coder84619 , do you see the version 1.0.7.1 on the stream deck store 
<img width="822" alt="image" src="https://user-images.githubusercontent.com/16235859/193978207-0ac9cf64-1e09-45cc-a1be-a32b390b3c3a.png">

Just restarting the stream deck app updates the plugins usually 

Same machine: 
<img width="474" alt="image" src="https://user-images.githubusercontent.com/16235859/193978399-3f05e74b-450e-432f-8a78-6d1cbde857cb.png">

Stream deck version : 
<img width="379" alt="image" src="https://user-images.githubusercontent.com/16235859/193978463-5d4c8e61-b5e4-4644-8e85-b35591d7b77f.png">

**Comment by Coder84619** [comment]
Yes, I saw the Oct 3 release which is why I jumped on it. I uninstalled/reinstalled 1.0.7.1 twice, and like I said, rebooted my Mac once too.

**Comment by SENTINELITE** [comment]
Hmm, @Coder84619. 

Let's see if we can get this sorted!

Can you try [these steps.](https://www.craft.do/s/SNZGwoLUs6uumS)

Let me know how that goes!

**Comment by Coder84619** [comment]
Still broke:

bundle.min.js:1 Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID
main_pi.js:6 Uncaught ReferenceError: Sentry is not defined
    at main_pi.js:6
main_pi.js:40 Uncaught ReferenceError: Cannot access 'uuid' before initialization
    at connectElgatoStreamDeckSocket (main_pi.js:40)
    at <anonymous>:1:1
main_pi.js:797 Uncaught ReferenceError: isSayvoice is not defined
    at toggleAccessNew (main_pi.js:797)
    at HTMLButtonElement.onclick (main_pi.html:87)
toggleAccessNew @ main_pi.js:797
onclick @ main_pi.html:87

**Comment by SENTINELITE** [comment]
Hey, @Coder84619! 

Looks like there's some sort of Network firewall, Ad blocker, VPN, etc, causing the socket to not properly communicate.

Can you try disabling them & see if that fixes it?

**Comment by Coder84619** [comment]
Ah so I found NextDNS was blocking: browser.sentry-cdn.com

I put that on the allow list and now the plugin loads. Thanks!

**Comment by SENTINELITE** [comment]
@Coder84619 

Glad it's working now! I need to find a way to detect if it's getting blocked, & workaround that, so the app doesn't refuse to work... 😊

Let me know if there's anything else you need/wish was better!

**Comment by Shellgio** [comment]
I still have this issue.

**Comment by SENTINELITE** [comment]
Hey, @bolincho! 

Can you join the [Discord](https://sentinelite.com/discord-shortcuts-support) server & try running V2 a try? It's got a wide array of improvements. (Albeit there's still bugs)
• Note you must get the proper role `StreamDeck-Shortcuts-Testers` in order to access the category.

This particular issue is "resolved", as there's no network stuff (currently).

Thanks!
